addons/mail/static/src/models/composer/composer.js

```diff
@@ -566,6 +566,32 @@ function factory(dependencies) {
             return [['unlink', unmentionedChannels]];
         }
 
+        /**
+         * Executes the given async function, only when the last function
+         * executed by this method terminates. If there is already a pending
+         * function it is replaced by the new one. This ensures the result of
+         * these function come in the same order as the call order, and it also
+         * allows to skip obsolete intermediate calls.
+         *
+         * @param {function} func
+         */
+        async _executeOrQueueFunction(func) {
+            if (this._hasMentionRpcInProgress) {
+                this._nextMentionRpcFunction = func;
+                return;
+            }
+            this._hasMentionRpcInProgress = true;
+            this._nextMentionRpcFunction = undefined;
```

Comment:
Would define them in `init()`. Easier to track special props of a class, which is super handful when refactoring.

Reply:
I guess you mean `_willCreate`

Reply:
yes